package.json

```diff
@@ -5,25 +5,30 @@
   "author": "Broadcom",
   "license": "EPL-2.0",
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "__tests__/__packages__/*"
   ],
   "scripts": {
     "build": "gulp updateLicense && lerna run build && npm run lint && npm run checkTestsCompile && npm run circularDependencyCheck",
+    "build:exe": "cd zowex && cargo build && cargo clippy && cargo test",
     "clean": "lerna run --parallel clean",
+    "clean:exe": "cd zowex && cargo clean",
     "installWithBuild": "npm install && npm run build",
     "checkTestsCompile": "echo \"Checking that test source compiles...\" && tsc --project __tests__/test-tsconfig.json --noEmit",
     "circularDependencyCheck": "lerna run --parallel circularDependencyCheck -- -- --warning --no-spinner",
     "lint": "eslint \"packages/**/*.ts\" \"**/__tests__/**/*.ts\"",
     "lint:packages": "eslint \"packages/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
     "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
-    "update:imperative": "npm i --package-lock-only --no-package-lock --save-exact @zowe/imperative@latest && syncpack fix-mismatches --dev --prod --filter @zowe/imperative && npm i",
+    "update:imperative": "npm i --package-lock-only --no-package-lock --save-exact @zowe/imperative@next && syncpack fix-mismatches --dev --prod --filter @zowe/imperative && npm i",
```

Comment:
Maybe you are just waiting for the imperative packageto publish, but the reference to @next in the update:imperative script seems like something that could be easy to overlook. So, I thought that I would mention it.